seeders/10_languages.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: queryInterface => {
+    const faker = require('faker');
+    const range = require('lodash/range');
+
+    const arr = range(1, 20).map((value, index) => ({
+      language: faker.address.country()
```

Comment:
you can use https://github.com/faker-js/faker#-localization instead of country

Reply:
faker.locale is for the language that we want for our fake data